tests/pos/i14177b.scala

```diff
@@ -0,0 +1,15 @@
+class T
+
+inline given fail1: T with
```

Comment:
Does transparent inline works / do anything useful here?

Reply:
It makes sure that we do not hit the previous issue where the `summonAll` was not inlined due to the `inline` flag that was kept by mistake on the `class fail1`.

Reply:
I mean, I there are no testcases with transparent in this pr

Reply:
Sorry, I completely missed the transparent. I will add some test cases.

Reply:
Transparent is failing for the wrong reason. I do not see how transparent would be useful here because we already have the precise type. I will disallow this combination and make sure the error message is correct.